pkg/infra/game/agent/random.go

```diff
@@ -24,9 +25,9 @@ func NewRandomAgent() *RandomAgent {
 
 func (r RandomAgent) HandleFightMessage(m message.TaggedMessage, view *state.View, agent BaseAgent, log *immutable.Map[commons.ID, decision.FightAction]) decision.FightAction {
 	fight := rand.Intn(3)
-
 	switch fight {
 	case 0:
+		agent.log(logging.Trace, logging.LogField{"bravery": r.bravery}, "Cowering")
 		return decision.Cower
 	case 1:
```

Comment:
maybe also add a log here for fight? 
```suggestion
	case 1:
		agent.log(logging.Trace, logging.LogField{"bravery": r.bravery}, "Fight")

```

Reply:
Could do, but it was just for an example. Up to agent teams as to what they want to want to log ig